cmd/translatesfx/translatesfx/testdata/sa-metrics-to-exclude-monitor.yaml

```diff
@@ -0,0 +1,14 @@
+---
+signalFxAccessToken: "abc123"
+signalFxRealm: us1
+
+intervalSeconds: 10
+
+logging:
+  level: info
+
+monitors:
+  - type: cpu
+    metricsToExclude:
```

Comment:
If we do start to use the agent config type directly in translation this will be a problem since it's not valid afaict.

Reply:
Noted, will look into it (it has worked in the tests I have run).